### PR TITLE
Chore/update confirmation modal

### PR DESCRIPTION
**Pull request by MildTomato**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES ~~/NO~~

## What kind of change does this PR introduce?

- `<ConfirmationModal/>` updated to use same styling as `<ConfrimTextModal/>`
- Also using similar prop names

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

Before
Screenshot 2024-03-29 at 3 19 09 PM

After
Screenshot 2024-03-29 at 3 19 23 PM

### CODE REVIEW

File: apps/studio/components/interfaces/Realtime/Inspector/RealtimeFilterPopover/index.tsx [hunk 1]
```diff
@@ -211,7 +215,13 @@ export const RealtimeFilterPopover = ({ config, onChangeConfig }: RealtimeFilter
           setApplyConfigOpen(false)
           setOpen(false)
         }}
-      />
+      >
+        <p className="text-sm text-foreground"></p>
```
Comment (joshenlim) [inline apps/studio/components/interfaces/Realtime/Inspector/RealtimeFilterPopover/index.tsx:219]: remove?

Comment (joshenlim) [inline apps/studio/components/interfaces/Realtime/Inspector/RealtimeFilterPopover/index.tsx:219]: remove?

```suggestion
```

File: packages/ui-patterns/Dialogs/ConfirmationModal.tsx [hunk 2]
```diff
@@ -1,80 +1,216 @@
-import { MouseEventHandler, PropsWithChildren, useEffect, useState } from 'react'
-import { Button, Modal } from 'ui'
+// import { MouseEventHandler, PropsWithChildren, useEffect, useState } from 'react'
```
Comment (joshenlim) [inline packages/ui-patterns/Dialogs/ConfirmationModal.tsx:1]: should we just remove all these? am okay with just leaving it if its temp

**Review by joshenlim — APPROVED** [review]

File: packages/ui-patterns/Dialogs/ConfirmationModal.tsx (hunk 2 above)
Comment (MildTomato) [reply packages/ui-patterns/Dialogs/ConfirmationModal.tsx]: fixe

File: apps/studio/components/interfaces/Realtime/Inspector/RealtimeFilterPopover/index.tsx (hunk 1 above)
Comment (MildTomato) [reply apps/studio/components/interfaces/Realtime/Inspector/RealtimeFilterPopover/index.tsx]: fixed

**Pull request merged**